doc/contributing/release_notes.rst

```diff
@@ -0,0 +1,13 @@
+How to write release notes
+==========================
+
+Release notes are generated by a script from individual changelog strings.
```

Comment:
It is applicable only for tarantool/tarantool repository, but the article is about style and I guess it should be general enough to apply for any project. I would not mention any technical details: generation scripts, file location, file formatting, etc; and would concentrate on the style here.

Reply:
@patiencedaur Please, respond at comments, when they're resolved. This one looks resolved now.